index.ts

```diff
@@ -3,14 +3,14 @@ import EfficientNetModel from "./src/EfficientnetModel";
 import EfficientNetResult from "./src/EfficientNetResult";
 import { EfficientNetCheckPoint } from "./src/EfficientNetCheckPoint";
 import {
-  EfficientNetLableLanguage,
+  EfficientNetLabelLanguage,
```

Comment:
thanks a lot ! need to refactor at `/playground` also

Reply:
do you mean in general or regarding the variable rename? cause I've tried searching for the typo and couldn't find it in that folder